sdks/python/apache_beam/runners/portability/flink_runner_test.py

```diff
@@ -189,12 +190,28 @@ def test_external_transforms(self):
                            value_deserializer='org.apache.kafka.'
                                               'common.serialization.'
                                               'LongDeserializer',
-                           expansion_service=expansion_address))
+                           expansion_service=get_expansion_service()))
       self.assertTrue('No resolvable bootstrap urls given in bootstrap.servers'
                       in str(ctx.exception),
                       'Expected to fail due to invalid bootstrap.servers, but '
                       'failed due to:\n%s' % str(ctx.exception))
 
+      # We just test the expansion but do not execute.
```

Comment:
FYI: @ihji is looking into adding a validates runner test suite for cross-language transforms that includes Kafka.

Reply:
Yes, I saw the PR (#8397). That will be great to extend our test coverage here.